src/ontology/analysis/analyze_mondo_stats/README.md

```diff
@@ -0,0 +1,307 @@
+- DOID	11390
+- ICD10CM	9142
+- NCIT	3804
+- OMIM	9333
+- Orphanet	11035
+- UMLS	20738
+
+
+### Alignment with the source - Number of terms from the sources that are in Mondo (as Mondo:equivalent)
+To generate these counts, follow the steps above for "Alignment with the source - Total number of diseases in sources", but create new queries where the xref must also have a source annotation of `MONDO:equivalentTo`.
+
+- Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_all_human_disease_mondo_classes_with_xref-equivalentTo.ru data/output/all_human_disease_mondo_class_xref_count-equivalentTo.tsv
+```
+
+
+#### Alignment to the source NCIT - Number of terms from the sources that are in Mondo (as Mondo:equivalent)
+- Query Mondo for a _list_ of all Mondo classes that are in the human disease branch, i.e. children of MONDO:0700096 'human disease' and have an xref to NCIT and also have an xref source annotation of `MONDO:equivalentTo`. Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_mondo_class_list_with_ncit_xref-equivalentTo.ru data/input/all_mondo_classes_with_ncit_xref-equivalentTo.tsv
+```
+
+- Check of overlapping NCIT CURIEs in each file by running these commands from the Terminal:
+```
+$ awk -F'\t' 'NR==FNR {xref[$1]; next} $3 in xref' data/input/ncit_neoplasm_children.tsv data/input/all_mondo_classes_with_ncit_xref-equivalentTo.tsv > data/output/all_mondo_classes_with_ncit_xref_neoplasm-branch-equivalentTo.tsv
+
+$ wc data/output/all_mondo_classes_with_ncit_xref_neoplasm-branch-equivalentTo.tsv
+    3452   18507  210236 data/output/all_mondo_classes_with_ncit_xref_neoplasm-branch-equivalentTo.tsv
+```
+
+#### Results for Alignment with the source - Number of terms from the sources that are in Mondo (as Mondo:equivalent)
+- DOID	11185
+- ICD10CM	1116
+- NCIT	3452
+- OMIM	9141
+- Orphanet	8176
+- UMLS	20738
+
+
+### Request 3: Externally Managed Content
+For Mondo terms are in the human disease branch, i.e. children of MONDO:0700096 'human disease', get the total number of diseases represented by each Externally Managed Content source (see below). This should be the count of distinct Mondo classes that contain at least one database_cross_reference to each of the "Externally Managed Content" sources. The query can be run against `mondo-edit.obo`, but must use the parameter `reason` with the robot command since many Mondo classes only classify with a parent of MONDO:0700096 'human disease' in the inferred hierarchy.
+
+- Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_all_human_disease_mondo_classes_with_EMC_xref.ru data/output/all_human_disease_mondo_class_xref_EMC_count.tsv
+```
+
+#### Externally Managed Content
+- GARD
+- MedGen
+- NANDO
+- NORD
```

Comment:
Shouldn't UMLS be included here?

Reply:
Sure, this might be a better place since that data is in the EMC pipeline. However, UMLS was listed as a Source twice in the [document](https://docs.google.com/document/d/1vPl2x8gfTypjzcb9ND8qXXfKIjPDE4RwuExmzAUpM6Q/edit?tab=t.0). Was there a reason for that?

Reply:
Just move UMLS to EMC for clarity